sql/2019/14_CMS/14_02.sql

```diff
@@ -0,0 +1,24 @@
+#standardSQL
+# 14_02: AMP plugin version
+SELECT
+  client,
+  amp_plugin_version,
+  COUNT(0) AS freq,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct
+FROM (
+  SELECT
+    client,
+    url,
+    REGEXP_EXTRACT(body, '(?i)<meta[^>]+name=[\'"]?generator[^>]+content=[\'"]?AMP Plugin v(\\d+\\.\\d+[^\'">]*)') AS amp_plugin_version
+  FROM
+    `httparchive.almanac.summary_response_bodies`
+  WHERE
+    firstHtml)
+JOIN
+  (SELECT _TABLE_SUFFIX AS client, url FROM `httparchive.technologies.2019_07_01_*` WHERE app = 'WordPress')
+USING
+  (client, url)
+GROUP BY
+  client,
+  amp_plugin_version
```

Comment:
1. Results are showing there are only 386 mobile websites out of 3 million wordpress sites which use this plugin... when the plugin has 400k installs. Looking closer it appears the meta generator tag only appears when the plugin is set to AMP-first mode.

    Is this enough data to draw any conclusions?

2. Will we be treating AMP versions like the following as being the same version, or different?

    `1.2.0; mode=standard; experiences=website`
    `1.2.0; mode=standard; experiences=website,stories`

Reply:
Definitely true. I chatted with @amedina about this and he agreed that there's not enough representation of the plugin in the dataset to draw meaningful conclusions.